src/main/java/org/folio/circulation/resources/CheckOutByBarcodeResource.java

```diff
@@ -69,7 +69,7 @@
 
 public class CheckOutByBarcodeResource extends Resource {
 
-  public static final String DATE_TIME_FORMAT = "yyyy-MM-dd'Z'";
+  public static final String DATE_TIME_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSSZ";
```

Comment:
@dmytrotur @kostyantyn-kh  Given that @opochepam has changed mod-calendar to use a date rather than date time, does this change still make sense?

Reply:
@marcjohnson-kint Yes, this change relates to mod-calendar response. The url parameter still is passed in date-only format.

Reply:
@dmytrotur Thank you for explaining the use of this constant